src/languageFacts/colors.ts

```diff
@@ -168,11 +172,15 @@ export const colors: { [name: string]: string } = {
 	yellowgreen: '#9acd32'
 };
 
+const colorsRegExp = new RegExp(`^(${Object.keys(colors).join('|')})$`, "i");
```

Comment:
That's a huge regex. We can just do a lookup in the colors object instead

Reply:
This should be checked ascii case insensitive : https://github.com/web-platform-tests/wpt/blob/master/css/css-color/parsing/color-invalid-named-color.html#L32

But maybe it's fine to be a little bit less strict here?

Reply:
Interesting, I didn't know that toLowerCase does that. We use over the place to do case insensitive comparisons.
I suggest to ignore that for now.